app/components/common/hero/index.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { cn } from '@/app/lib/utils';
+
+interface Prop {
+  isCarousel?: boolean;
+  children: React.ReactNode;
+}
+
+export default function Hero({ children, isCarousel = true }: Prop) {
+  return (
+    <div
+      className={cn(
+        'relative overflow-hidden',
+        isCarousel ? 'h-[447px]' : 'h-[339px]',
+      )}
+    >
+      {children}
+      <div className="bg-bgBlack absolute bottom-0 w-full max-h-[40px] h-[15%] z-10 rounded-t-[22px]"></div>
```

Comment:
```suggestion
      <div className="bg-bgBlack absolute bottom-0 w-full max-h-10 h-[15%] z-10 rounded-t-[22px]"></div>
```

대신 arbitrary value를 사용하신 이유가 있나요?